alembic/env.py

```
@@ -19,6 +19,30 @@
 # Load environment variables from .env file
 load_dotenv()
 
+# Import the new config system and get database URL
+# ↳ This integrates with the unified config system
+from readwise_vector_db.config import settings, DatabaseBackend
```

Ruff (0.11.9)

Module level import not at top of file

(E402)